.github/dependabot.yml

```
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: daily
```

```suggestion
      interval: weekly
```
Weekly is good enough for these.

---

@ludeeus But why? If it can be daily, why not?

---

It doesn't cost anyone resources if there is no update available.

---

Because someone has to review these